testing/logging/test_reporting.py

```diff
@@ -1098,6 +1098,119 @@ def test_foo():
     )
 
 
+def test_cli_level_color_disabled_1(pytester: Pytester) -> None:
```

Comment:
Please use `pytest.mark.parmetrize` to merge your four tests into one, or alternatively create a private function in this module that calls pytest.maskepyfile to avoid a bit of code duplication.